app/views/govuk_component/metadata.raw.html.erb

```diff
@@ -9,55 +9,102 @@
 
   direction_class = ""
   direction_class = " direction-#{direction}" if local_assigns.include?(:direction)
+
+  metadata_items = []
+
+  if from.any?
+    metadata_items << [
+      t("govuk_component.metadata.from", default: "From"),
+      from
+    ]
+  end
+
+  if part_of.any?
+    metadata_items << [
+      t("govuk_component.metadata.part_of", default: "Part of"),
+      part_of
+    ]
+  end
+
+  if local_assigns.include?(:history)
+    metadata_items << [
+      t("govuk_component.metadata.history", default: "History"),
+      [history]
+    ]
+  end
+
+  if local_assigns.include?(:first_published) && first_published
+    metadata_items << [
+      t("govuk_component.metadata.published", default: "Published"),
+      [first_published]
+    ]
+  end
+
+  if local_assigns.include?(:last_updated) && last_updated
+    if local_assigns.include?(:see_updates_link)
+      update_text = t("govuk_component.metadata.see_all_updates", default: "see all updates")
+    end
+
+    metadata_items << [
+      t("govuk_component.metadata.last_updated", default: "Last updated"),
+      [last_updated],
+      update_text || nil
+    ]
+  end
+
+  if other.present?
+      other.each do |title, definition|
+        definition ||= []
+        definition = Array(definition)
+        if definition.any?
+          metadata_items << [
+            title,
+            definition
+          ]
+        end
+      end
+  end
 %>
 <div class="govuk-metadata<%= direction_class %>" data-module="toggle">
   <dl>
-    <% if from.any? %>
-      <dt><%= t("govuk_component.metadata.from", default: "From") %>:</dt>
-      <dd>
-        <%= render file: 'govuk_component/metadata_sentence.raw', locals: { items: from, toggle_id: "from" } %>
-      </dd>
-    <% end %>
-    <% if part_of.any? %>
-      <dt><%= t("govuk_component.metadata.part_of", default: "Part of") %>:</dt>
-      <dd>
-        <%= render file: 'govuk_component/metadata_sentence.raw', locals: { items: part_of, toggle_id: "part-of" } %>
-      </dd>
-    <% end %>
-    <% if local_assigns.include?(:history) %>
-      <dt><%= t("govuk_component.metadata.history", default: "History") %>:</dt>
-      <dd><%= history %></dd>
-    <% end %>
-    <% if local_assigns.include?(:first_published) && first_published %>
-      <dt><%= t("govuk_component.metadata.published", default: "Published") %>:</dt>
-      <dd><%= first_published %></dd>
-    <% end %>
-    <% if local_assigns.include?(:last_updated) && last_updated %>
-      <dt><%= t("govuk_component.metadata.last_updated", default: "Last updated") %>:</dt>
+    <% metadata_items.each_with_index do |(title, items, extratext), index| %>
```

Comment:
I wrote it that way initially by @nickcolley suggested the markup should be in the view. We could always amend it to that in future if needed?